src/deprecation.h

```diff
@@ -10,16 +10,7 @@
 // * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
 static const int APPROX_RELEASE_HEIGHT = 632000;
 static const int WEEKS_UNTIL_DEPRECATION = 16;
-// static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 24);
-
-// TEMPORARY!
-// This is hardcoded to a height that is roughly 10 weeks following the
-// activation of Blossom on mainnet, which is itself almost 6 weeks following
-// this release. This is done because the calculation of the deprecation height
-// does not account for the change in block target spacing; after Blossom has
-// activated, this should be removed and the previous line fixed to use
-// APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48)
-static const int DEPRECATION_HEIGHT = 734240;
+static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48);
```

Comment:
Normally people building from master between releases X and Y would have EoS halts at the same time as X. This PR has the side-effect of setting `DEPRECATION_HEIGHT` to 761024‬, which is roughly 23 days after 2.1.0 EoS halt. However, as this falls inside the 6-week release cycle that we don't go below, we still ensure that 2.1.1 is active at the time builds from master between 2.1.0 and 2.1.1 would reach EoS halt (not that we support production builds from master, but the technical shutdown still occurs). tl;dr I think this is fine.

Reply:
I verified the figures here and I agree, this would be fine to merge with or without any delay. I think we should just merge it now to get it out of the way.